packages/crypto/browser.config.mts

```diff
@@ -1,7 +1,10 @@
 /// <reference types="vitest" />
 import { defineConfig } from 'vite';
+import { nodePolyfills } from 'vite-plugin-node-polyfills';
 
 export default defineConfig({
+  // fix error - Buffer is not defined
+  plugins: [nodePolyfills()],
```

Comment:
Why is this needed if we have the "buffer" package in our package.json?

Reply:
fix  `Buffer is not defined` when i start test